kclvm/sema/src/resolver/arg.rs

```diff
@@ -7,21 +7,40 @@ use std::rc::Rc;
 use crate::resolver::pos::GetPos;
 
 impl<'ctx> Resolver<'ctx> {
+    fn get_func_name(&mut self, func: &ast::Expr) -> String {
+        let mut callee = func;
+        loop {
+            match callee {
+                ast::Expr::Identifier(identifier) => {
+                    return format!("\"{}\"", identifier.get_name());
+                }
+                ast::Expr::Selector(selector_expr) => {
+                    return format!("\"{}\"", selector_expr.attr.node.get_name());
+                }
+                ast::Expr::Lambda(_) => return "anonymous function".to_string(),
+                ast::Expr::Paren(paren_expr) => callee = &paren_expr.expr.node,
+                _ => bug!("invalid callee : {:?}", callee),
```

Comment:
With the change in syntax, it may not be possible to enumerate all function forms. It is recommended to return "function" here or remove the `get_func_name` calling directly.

Reply:
now we just return "anonymous function" to unmatched function forms